docs/src/Text.doc.js

```diff
@@ -155,22 +150,22 @@ card(
 <Box maxWidth={240}>
   <Box marginBottom={2}>
     <Text weight="bold">normal:</Text>
-    <Text overflow="normal" leading="tall">
```

Comment:
@AlbertCarreras could you also write a quick codemod to remove `leading` from `Text`? That way other users of gestalt can also easily upgrade.

Reply:
DONE